llpc/translator/lib/SPIRV/SPIRVToLLVMDbgTran.cpp

```diff
@@ -167,7 +167,13 @@ DIType *SPIRVToLLVMDbgTran::transTypePointer(const SPIRVExtInst *DebugInst) {
   DIType *PointeeTy = nullptr;
   if (BM->getEntry(Ops[BaseTypeIdx])->getOpCode() != OpTypeVoid)
     PointeeTy = transDebugInst<DIType>(BM->get<SPIRVExtInst>(Ops[BaseTypeIdx]));
+#if LLVM_MAIN_REVISION && LLVM_MAIN_REVISION < 444152
```

Comment:
I think the exact revision number to use here might be:
```
$ git rev-list --count bac974278c5e5a3d6dea40d2d22cb36bcc487cee
443989
```

Reply:
The upcoming merge will go as far as df415774d2a5.
```
$ git rev-list --count df415774d2a5
444152
```